xds/src/test/java/io/grpc/xds/XdsLbStateTest.java

```diff
@@ -172,7 +174,7 @@ public void setUp() throws Exception {
     doReturn("fake_authority").when(helper).getAuthority();
     doReturn(mock(ChannelLogger.class)).when(helper).getChannelLogger();
     lbRegistry.register(childLbProvider);
-    localityStore = new LocalityStoreImpl(helper, interLocalityPickerFactory, lbRegistry);
+    localityStore = new LocalityStoreImpl(helper, interLocalityPickerFactory, lbRegistry, random);
```

Comment:
As random is not called in this test, and it's not stubbed, I would add a `verifyZeroIteractions(random)` in `@After` so that we know the default return value 0 didn't affect the test in any way.

Reply:
Done.